app.js

```diff
@@ -40,13 +38,19 @@ function randomImage() {
 let image = randomImage();
```

Comment:
You're going to want your randomImage function to interact directly with your objects.